<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attendance</class>
 <widget class="QWidget" name="attendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>709</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:  rgb(42, 131, 162)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{color: rgb(0,0,0);
border-image: url(:/new/res/43.gif);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;
}
QPushButton:hover{
background-color: rgb(22, 94, 131);
border-image: url(:/new/res/46.gif);color: rgb(0, 255, 255);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;}
QPushButton:pressed{
border-image: url(:/new/res/46.gif);color: rgb(0, 255, 255);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;}
</string>
    </property>
    <property name="text">
     <string>开始点名</string>
    </property>
   </widget>
   <widget class="QLabel" name="lesson_namelabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;设置视频流地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/res/设置 (2).png)}QPushButton:hover{border-image: url(:/new/res/设置 (3).png)}
QPushButton:pressed{border-image: url(:/new/res/设置 (3).png)}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="videolabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>701</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/res/Rectangle 5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>391</width>
      <height>591</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>170</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>170</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{color: rgb(0,0,0);
border-image: url(:/new/res/43.gif);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;
}
QPushButton:hover{
background-color: rgb(22, 94, 131);
border-image: url(:/new/res/46.gif);color: rgb(0, 255, 255);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;}
QPushButton:pressed{
border-image: url(:/new/res/46.gif);color: rgb(0, 255, 255);
font: 25 12pt &quot;微软雅黑 Light&quot;;
border:2px groove gray;
border-radius:15px;
padding:4px 4px;}
</string>
    </property>
    <property name="text">
     <string>结束点名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_indexButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;返回主页&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/res/主页 (1).png)}QPushButton:hover{border-image: url(:/new/res/主页.png)}
QPushButton:pressed{border-image: url(:/new/res/主页.png)}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../photo.qrc"/>
 </resources>
 <connections/>
</ui>
